lib/mongo/events.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Mongo.Events.ServerDescriptionChangedEvent do
```

Comment:
> I'm assuming you wanted me to use records base on the assumption that it's internal-only, which isn't true, so I'll leave it as structs unless you have any objections.

Correct, sounds good. I would still suggest hiding the calls to `GenEvent.notify` behind an API on `Mongo.Events` though.